### PR TITLE
1011 make existing charts work with Mobile view

### PR DESCRIPTION
**Pull request by mmnoo**
⚠️ ⚠️ ⚠️ NOTE TO SELF: CHANGE BASE BRANCH TO DEVELOP BEFORE MERGE ⚠️ ⚠️ ⚠️

[Trello card](https://trello.com/c/Cnt8XPkw/1011-dashboard-visualizations-aggregate-translate-plotly-code-into-javascript-for-aggregate-plots-see-attached-file?filter=member:mnunes24)

Steps to test:
- clone `https://github.com/data-mermaid/mermaid-dash-v2`
- follow readme to get server running (api is prod, so dont need local back end)
- I updated the MERMAID secure note in dashlane. You will find env vars for this app at bottom of it
- load the app in the browser
- view the 4 new charts in desktop view
- shrink the screen to mobile size, the metrics pane is collapsed at the bottom of the screen
- expand the metrics pane to view the charts in mobile view

Screenshot 2024-12-18 at 1 19 03 PM
Screenshot 2024-12-18 at 1 09 49 PM

### CODE REVIEW

**Comment by mmnoo** [comment]
@alanjleonard this branch is ready for design review. If merged its best to review it on `develop`, otherwise you can use the feature branch.

**Review by alanjleonard — CHANGES_REQUESTED** [review]
- Unable to scroll past charts on touch devices since it zooms into the chart. Could probably just disable the zoom feature altogether.
- Charts with data extend off the end of the screen. (Chrome on Android, screen recording shows both issues)

https://github.com/user-attachments/assets/fc104681-9cd1-4970-b773-6245141828d7

**Review by alanjleonard — CHANGES_REQUESTED** [review]
1. Use either the comma separator for number of surveys/colonies or abbreviate it using k to represent 1,000. So this `267357 Colonies` turns into this `267,357 Colonies` or this `267k Colonies`. Which ever is easier for you. I think we have a function in GFCR somewhere to add the thousands separator. You can ask Julia.
2. Prevent the buttons from shifting when switching between tabs.
3. On hover, give the inactive tab a border and the secondary button hover color. 
image
4. Remove the gap at the bottom of the metric pane. 
image

File: src/components/MetricsPane/MetricsPane.jsx [hunk 1]
```diff
@@ -152,57 +154,62 @@ const MetricsPane = ({
...
+      <ChartsWrapper $showMobileExpandedMetricsPane={showMobileExpandedMetricsPane}>
+        <MetricsPaneChartTabs
+          id="hard-coral-cover"
+          aggregatePanelContent={<AggregateHardCoralCover />}
+          timeSeriesPanelContent={<div style={{ height: '270px' }}>Coming soon</div>}
+        />
+        <MetricsPaneChartTabs
+          id="fish-biomass"
+          aggregatePanelContent={<AggregateFishBiomass />}
+          timeSeriesPanelContent={<div style={{ height: '270px' }}>Coming soon</div>}
```
Comment (julia-rom) [inline src/components/MetricsPane/MetricsPane.jsx:198]: could simplify the place holder:

`const ComingSoonPlaceholder = () => <div style={{ height: '270px' }}>Coming soon</div>`

**Review by julia-rom — APPROVED** [review]

**Pull request merged**